packages/middleware-render-error-info/test/unit/lib/index.spec.js

```diff
@@ -64,6 +64,9 @@ describe('@dotcom-reliability-kit/middleware-render-error-info', () => {
 		let response;
 
 		beforeEach(() => {
+			// The error page template contains the current year
+			jest.useFakeTimers().setSystemTime(new Date('1888-01-09'));
```

Comment:
`1888-01-09`

> 9 January 1888 - The Financial Times Launches

😁

Reply:
I had no idea that's why the 1888 restaurant is called the 1888 restaurant until today 🙈